tests/test_sandbox.py

```diff
@@ -25,47 +25,125 @@
 from django.test import TestCase
 from django.urls import reverse
 from .base_test_mixins import SingleCourseTestMixin
-from course.models import Participation
-from course.constants import participation_permission as pperm
 
+QUESTION_MARKUP = """
+type: TextQuestion\r
+id: half\r
+value: 5\r
+prompt: |\r
+    # A half\r
+    What's a half?\r
+answers:\r
+    - type: float\r
+      value: 0.5\r
+      rtol: 1e-4\r
+    - <plain>half\r
+    - <plain>a half
```

Comment:
Are these `\r`s necessary?

Reply:
Seems to be. I remembered without them the tests failed with bad formatting. I need to test again to make sure.

Reply:
You're right, those trailing string are not necessary.